data/names/ru.json

```diff
@@ -3447,31 +3535,50 @@
       "Винсон",
       "Винтер",
       "Витале",
-      "Виттен",
       "Витт",
+      "Виттен",
       "Вишневски",
+      "Вишневский",
+      "Вмятина",
+      "Во",
       "Воган",
+      "Волк",
       "Вольф",
-      "Вонг",
+      "Воля",
+      "Вомак",
       "Вон",
+      "Вонг",
+      "Ворота",
       "Восс",
-      "Во",
+      "Восток",
+      "Вред",
```

Comment:
This is a translation of either Harm or Hurt as a common noun (as in injury). It would be transcribed either Харм (as in [Харм Кёйперс](https://ru.wikipedia.org/wiki/%D0%9A%D1%91%D0%B9%D0%BF%D0%B5%D1%80%D1%81,_%D0%A5%D0%B0%D1%80%D0%BC), the Dutch skater) or Хёрт (as in [Джон Хёрт](https://ru.wikipedia.org/wiki/%D0%A5%D1%91%D1%80%D1%82,_%D0%94%D0%B6%D0%BE%D0%BD), the actor).

Reply:
It's `Harm`. I suppose the transcription would be `Хармс` then?

Reply:
Like [Даниил Хармс](https://ru.wikipedia.org/wiki/%D0%A5%D0%B0%D1%80%D0%BC%D1%81,_%D0%94%D0%B0%D0%BD%D0%B8%D0%B8%D0%BB_%D0%98%D0%B2%D0%B0%D0%BD%D0%BE%D0%B2%D0%B8%D1%87), yup